servicemesh-install/topics/tutorial-prometheus.adoc

```diff
@@ -4,6 +4,7 @@
 TODO
 Need a short overview of Prometheus.
 ////
+This tutorial uses {ProductShortName} and the `bookinfo` tutorial to demonstrate how you can query for metrics using Prometheus.
 
 Prerequisites:
 
```

Comment:
Line 55 -- you have odd line breaks here (after name}') that means the commands don't copy/paste correctly.

oc get prometheus -n istio-system -o jsonpath='{.items[*].spec.metrics[*].name}'
requests_total request_duration_seconds request_bytes response_bytes tcp_sent_bytes_total tcp_received_bytes_total[root@slareceived_bytes_totalrequests_total request_duration_seconds request_bytes response_bytes tcp_sent_bytes_total tcp_ 
bash: requests_total: command not found...

Reply:
Good catch, thanks!